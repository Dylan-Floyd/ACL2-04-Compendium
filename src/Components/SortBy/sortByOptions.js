export default {
  Name: 'pokemon',
  Height: 'height',
  Weight: 'weight',
  Type: 'type',
  Attack: 'attack',
  Defense: 'defense',
  HP: 'hp'
}