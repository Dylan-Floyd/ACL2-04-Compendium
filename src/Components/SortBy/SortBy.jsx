import React from 'react'
import Select from '../Select/Select.jsx'
import sortByOptions from './sortByOptions.js'

export default function SortBy({
  selectedOption,
  handleChange
}) {
  return (
    <Select
      label='Sort By: '
      selectedOption={selectedOption}
      optionsObject={sortByOptions}
      handleChange={handleChange}
    />
  )
}
