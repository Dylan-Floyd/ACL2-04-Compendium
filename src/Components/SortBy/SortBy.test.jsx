import { render } from '@testing-library/react'
import SortBy from './SortBy.jsx'

it('Should render the SortBy component', () => {
  const { container } = render(<SortBy />)
  expect(container).toMatchSnapshot()
})