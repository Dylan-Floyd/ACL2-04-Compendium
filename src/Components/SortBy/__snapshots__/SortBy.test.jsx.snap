// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the SortBy component 1`] = `
<div>
  <label>
    Sort By: 
    <select>
      <option
        value="pokemon"
      >
        Name
      </option>
      <option
        value="height"
      >
        Height
      </option>
      <option
        value="weight"
      >
        Weight
      </option>
      <option
        value="type"
      >
        Type
      </option>
      <option
        value="attack"
      >
        Attack
      </option>
      <option
        value="defense"
      >
        Defense
      </option>
      <option
        value="hp"
      >
        HP
      </option>
    </select>
  </label>
</div>
`;
